README.md

```diff
@@ -27,6 +27,12 @@ Build
 -----------------
 ### Linux
 
+Update System:
+```{r, engine='bash'}
+sudo apt-get update
+sudo apt-get upgrade
+```
```

Comment:
Why is this here? Reference Issue #27 

Please keep code that belongs to issues in separate PR's and don't mix code that belongs to a feature with issue fixes

Reply:
Will do in the future.